package.json

```diff
@@ -20,6 +20,8 @@
   "author": "GitHub",
   "license": "CC0-1.0",
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
     "electron": "^9.0.4",
     "tslint": "^6.1.2",
```

Comment:
Remove `tslint` from the `packages.json` file as it is no longer used

Reply:
fixed it, it was done, but got reverted when merged from master.